docs-markdown/src/controllers/cleanup/cleanup-controller.ts

```diff
@@ -195,10 +195,9 @@ export async function applyCleanupFolder(uri: Uri) {
 				const promises: Promise<any>[] = [];
 				// check for dirty files
 				files.map(file => {
-					const fileName = basename(file);
-					const modifiedUri = join(uri.path, fileName).replace(/\\/g, '/');
-					if (extname(modifiedUri) === '.md') {
-						workspace.openTextDocument(Uri.parse(modifiedUri)).then(doc => {
+					file = file.replace(/\\/g, '/');
```

Comment:
I stumbled upon what I *think* may be a bug.  When we have nested folders in the Uri we pass to  `applyCleanupFolder` the `modifiedUri` ends up with the wrong path to the file, so it never gets cleaned up. For example, in the screenshot below `/test` is stripped from the `modifiedUri`, which results in a file not found exception when running the tests 

<img width="809" alt="cleanup bug" src="https://user-images.githubusercontent.com/46545485/83591014-d8475c00-a50b-11ea-80dd-203be7a96f37.png">

In addition to this fix I have modified one test case and added a new one.  I've modified the existing test to make sure the we can recursively cleanup folders (the issue I'm fixing).  The new test is just adding some test coverage for dirty files.  I noticed most of these tests are just expecting method calls but these new tests will verify the output which I think strengthens the test coverage.

Reply:
Sounds good, the new test looks good to me.